cvat-canvas3d/src/typescript/canvas3dController.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Canvas3dModel, Mode, Configuration } from './canvas3dModel';
```

Comment:
'Configuration' is declared but its value is never read.ts(6133)
Configuration not found in './canvas3dModel'eslintimport/named

Reply:
Resolved